libs/coin-modules/coin-polkadot/src/network/node/stakingInfo.ts

```diff
@@ -23,10 +24,13 @@ export const fetchStakingInfo = async (
   const historicApi = await api.at(hash);
 
   if (currency?.id === "assethub_polkadot" && !historicApi.query.staking) {
+    const assethubConfig = coinConfig.getCoinConfig(currency);
+
+    if (assethubConfig.hasBeenMigrated)
+      throw new Error("Staking info not available for assethub_polkadot");
```

Comment:
Is this exception caught somewhere ? we do not want the user to encounter a red cross on sync